### ISSUE TITLE
Request: prefix also for optiongroup

### ISSUE THREAD

**Issue by badulesia** [thdoan/pretty-dropdowns#8]
Hello.
You can set a prefix for an `<option>`, so you can display an icon (as example in the demo).
Is it possible to implement the same feature for an `<optgroup>`, using the same data-prefix way ?
In my case, I would like to display a country flag icon
Thank you

**Comment by thdoan** [comment]
Hi, thanks for the suggestion -- great idea! I'll add it to the next release.

**Comment by thdoan** [comment]
This feature is now available in [v4.12.0](https://github.com/thdoan/pretty-dropdowns/releases/tag/v4.12.0).

**Comment by badulesia** [comment]
about <optgroup> with prefix.
- a simple text is well displayed 
- if I use an <img>, it is correctly displayed when the dropdown is opened, but the first image stays when the dropdown is closed ! I have also try to embed the <img> into a <span>, same result.
![bad_icons](https://user-images.githubusercontent.com/3285335/38887609-9a226b78-4279-11e8-9b60-d04e3447e4bd.jpg)

**Comment by thdoan** [comment]
Thanks for reporting this. I'll roll out a fix ASAP...

**Comment by badulesia** [comment]
Oups ... right, some code were stripped.
Try this for example
`<optgroup data-prefix="<img src='someimage.png'>" label="sometext">`
`<option value="0" selected="selected">option0text</option>`
`</optgroup>`

The `<optgroup>` image and text are well displayed when the menu is opened, but the image stays when the menu is closed. If I insert several `<optgroup>`, it's the image from the first one that stays.

BTW, in your code, you can use a faster method to recover data attribute
instead of elOpt.getAttribute('data-prefix')
use elOpt.dataset.prefix
[documentation](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset)
This is now accepted by every web browsers (but not IE<11)

Thank you

**Comment by thdoan** [comment]
@badulesia thanks for the updated code and protip 👍

**Comment by badulesia** [comment]
Hello.
I have done some extra tests: if I added the same prefix (with image) to both the options and optgroup, the menu is well displayed when opened and closed. If no prefix in options, the menu displays the first optgroup prefix when closed. Can this be an issue while rendering the menu closed at launch ?
I'm trying to investigate the code ...

**Comment by badulesia** [comment]
Hello.
I have found a solution : no change in js code , just a css rule ! 😁 
```
.prettydropdown > ul:not(.active) > li:not(.selected):first-child {
  visibility: hidden; /* Prevent FOUC */
}
```
The previous rule `color:transparent` only disables text to be seen. This new rule disables everything yet preserving the shape and size. Simple, obvious and effective !
I have tested it with all the dropdown types (simple, multiple, with optgroups) and it seems to work ok.

B.
